apps/web/components/settings/platform/hooks/usePlatformMe.ts

```diff
@@ -6,14 +6,26 @@ export const usePlatformMe = () => {
   const QUERY_KEY = "get-platform-me";
   const platformMeQuery = useQuery<UserResponse>({
     queryKey: [QUERY_KEY],
-    queryFn: async (): Promise<UserResponse> => {
+    retryOnMount: false,
+    staleTime: 300000,
```

Comment:
300000 seconds is 83 hours, so i am wondering if someone goes to dashboard and then buys platform plan will they be able to use platform?

I would reduce this to an hour or something or maybe even 30 minutes.

Reply:
and in such a case remove retryOnMount